packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -114,11 +156,34 @@ export default class Meeting extends StatelessSparkPlugin {
     }
   }
 
+  /**
+   * Sets the first locus info on the class instance
+   * @param {Object} locus
+   * @returns {null}
+   * @memberof Meeting
+   */
+  parseLocus(locus) {
+    if (locus) {
+      this.locusUrl = locus.url;
+      if (locus.info) {
+        this.type = MeetingUtil.getLocusType(locus.info.locusTags);
+      }
+      if (locus.participants && locus.self) {
+        this.partner = MeetingUtil.getLocusPartner(locus.participants, locus.self);
+      }
+      if (this.partner) {
+        this.sipUri = this.partner.person.sipUrl || this.partner.person.emailAddress || this.partner.person.email;
+      }
+    }
+    this.setLocus(locus);
```

Comment:
Why do we call set locus outside of the if(locus) block? Seems like this would/could fail if locus is null in the `setLocus` method

Reply:
I'll take a look.

Reply:
I can't recall why I did that but I changed it.